post-microsservice/src/test/kotlin/com/example/postmicrosservice/repository/PostRepositoryTest.kt

```diff
@@ -0,0 +1,49 @@
+package com.example.postmicrosservice.repository
+
+import com.example.postmicrosservice.model.Post
+import com.example.postmicrosservice.model.PostBuilder
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import kotlin.random.Random
+
+@SpringBootTest
+class PostRepositoryTest {
+    @Autowired
+    lateinit var postRepository: PostRepository
+
+    @BeforeEach
+    fun setup() {
+        postRepository.deleteAll()
```

Comment:
Why is it needed to delete all post repository before each setup?

Reply:
Because it's a test repository. So, before each test we need the DB to be clear for test execution.